rotkehlchen/tests/unit/accounting/test_exchanges.py

```diff
@@ -38,3 +42,75 @@ def test_account_for_coinbase_ledger_actions(rotkehlchen_api_server_with_exchang
     assert events_map[AccountingEventType.FEE] == 2
     assert events_map[AccountingEventType.ASSET_MOVEMENT] == 3
     assert sum(events_map.values()) == 11  # processed events
+
+
+@pytest.mark.parametrize('default_mock_price_value', [FVal(1.5)])
+@pytest.mark.parametrize('ethereum_accounts', [[]])
+def test_exchanges_removed_api_keys(rotkehlchen_api_server):
+    """
+    Test that if actions made on an exchange are stored in the DB but the API keys were removed
```

Comment:
This is not doing what it says. You should add the exchange, add the trades and then remove it. I know the result is gonna be the same but this is what you say you are testing, but this is not what the test is doing which is a problem.